SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPendingNetGame.h

```diff
@@ -19,4 +19,9 @@ class USpatialPendingNetGame : public UPendingNetGame
 
 	virtual void SendJoin() override;
 
+#if ENGINE_MINOR_VERSION >= 25
+	//Made virtual in PendingNetGame.h, make it do nothing, as it would usually send a packet
+	virtual void SendInitialJoin() override {}
```

Comment:
I'm not even sure if we need this class anymore. This used to exist for client travel back when we had a different implementation of instansiating the spatial net driver.

Reply:
Removed my modification here, as it does not seem needed anymore, created ticket to remove SpatialPendingNetGame entirely: https://improbableio.atlassian.net/browse/UNR-3920.